src/test/kotlin/io/github/rybalkinsd/kohttp/interceptors/LoggingInterceptorTest.kt

```diff
@@ -55,4 +55,16 @@ class LoggingInterceptorTest {
         assertEquals(200, response.code())
     }
 
+    @Test
+    fun `curl command logging happens without exceptions `() {
```

Comment:
This test's output is below.
```
GET 200 - 1118ms https://postman-echo.com/get
╭--- cURL command -------------------------------
curl -X GET "https://postman-echo.com/get"
╰--- (copy and paste the above line to a terminal)
```

And a test result in the case of LoggingFormat.HTTP is below.
```
POST 200 - 2076ms http://postman-echo.com/post
--80d27159-868d-4252-8b65-392dd2596483
Content-Disposition: form-data; name="file"; filename="cat.gif"
Content-Length: 1046038

	"#+'!5+!95/=1"F;1H7%LB4LG@T��YL?ZH3]SE^YThUAh`Wm]HpbRqh`t��xaHxtp}nc}ys~jS~mZ�iK�|u�u`��7�{m�w^��~�z^��s��e�����{��g��F��r�����{�����������]������¸�Ʋ��þ�����|�ɡ���������������!�NETSCAPE2.0   !�
��-"��Q�]t�ZB��C���x�q�w#�4�
qߋR9�
M!
?�8�+[򒜂+��I�d"F�P��seiqSXa�b���=e�e(��c�/���z骎�j�*�6���eO�v��a �����w���de�\��>YU:��A�4��z?�8� Z�ϸߓ�
�^�'+
���
-c�f"4��oG�ID�9[\�9��� h� 07%�HڥX堟�	M]n�p�P:t�r^�- BI$�F��{
......... skip the rest
```